packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -91,11 +107,11 @@ def initialize_settings(self):
         self.settings["ai_default_tasks"] = default_tasks
         self.log.info("Registered all default tasks.")
 
-        if ChatOpenAINewProvider.auth_strategy.name not in os.environ:
-            raise EnvironmentError(f"`{ChatOpenAINewProvider.auth_strategy.name}` value not set in environment. For chat to work, this value should be provided.")
+        #if ChatOpenAINewProvider.auth_strategy.name not in os.environ:
+        #    raise EnvironmentError(f"`{ChatOpenAINewProvider.auth_strategy.name}` value not set in environment. For chat to work, this value should be provided.")
 
         ## load OpenAI provider
-        self.settings["openai_chat"] = ChatOpenAIProvider(model_id="gpt-3.5-turbo")
+        #self.settings["openai_chat"] = ChatOpenAIProvider(model_id="gpt-3.5-turbo")
```

Comment:
Think these can be fully deleted; don't think we'll ever backpedal on model configurability.